src/styles/theme.ts

```diff
@@ -1,4 +1,6 @@
 import { createTheme, Shadows } from '@mui/material/styles'
+import type {} from '@mui/x-date-pickers/themeAugmentation'
```

Comment:
This doesn't seem to be used

Reply:
I included it in case we want to [customise anything about the `<DatePicker>`](https://mui.com/x/react-date-pickers/getting-started/#typescript) in the future.